src/types/components.d.ts

```diff
@@ -0,0 +1,41 @@
+import { DefineComponent, PropType } from "vue";
+import { FormArraySchema, FormObjectSchema } from "./schema";
+
+type ClassBindingValue = string | Record<string, boolean>;
+type ClassBindingExpression =
+  | ClassBindingValue
+  | (string | Record<string, boolean>)[];
+
+declare const SchemaForm: DefineComponent<{
+  schema: {
+    type: PropType<FormObjectSchema | FormArraySchema>;
+    default: any;
+  };
+  sharedConfig: {
+    type: ObjectConstructor;
+    default: any;
+  };
+  preventModelCleanupOnSchemaChange: {
+    type: PropType<boolean>;
+    default: boolean;
+  };
+  nestedSchemaModel: {
+    type: PropType<string>;
+    default: string;
+  };
```

Comment:
This really is private API, maybe we should remove it from the types

Reply:
Oh, didn't realize that. Yea we should probably remove it.